### ISSUE TITLE
cifar networks as in DeepFault paper

### ISSUE THREAD

**Issue by MatoFD** [hasanferit/DeepFault#6]
Hello,

While trying to replicate the results of DeepFault paper with my experiments, I cannot replicate cifar networks. I can only find cifar10_test_model_1_512_leaky_relu, are the other two (2_256 and 4_128) in the repository and I missed them? If not, do you still have them and is it possible to upload them?

I tried training them with train_cifar_nn.py but noticed that when I try to use them they give really bad results, i.e. I train until 70% acc but when used over the whole test set they show <60% acc instead. I suspect this is becuase in train_cifar_nn both training and test inputs are normalized (x_train /= 255, x_test /= 255) but in run.py the inputs are used without normalizing. Is this intended?

**Comment by hfeniser** [comment]
Hi @MatoFD,

Unfortunately, I dont have those additional models. As you did, one can modify `train_cifar_nn.py` script to train those models. I would expect a lower accuracy (but not too low) with the modified models when compared to the original 1_512 model.

Yes, there is normalization incompatibility between the training script and the deepfault code. I will correct it soon. For a correct analysis, either both have to be normalized or both have to be non-normalized.

**Comment by MatoFD** [comment]
Thank you for your answer.

**Comment by hfeniser** [comment]
Of course, I close this issue now. If there is further inquiry please re-open it.
